### DISCUSSION TITLE
Support hex color in `Badge` color

### DISCUSSION THREAD

**Discussion by imranbarbhuiya** [mantinedev/mantine#2513] [Feature requests]
Currently badge color only supports `theme.keys`. But we need 10 more colors and we don't want to add all the colors to theme just for one use. Adding support for hex code will help a lot.

**Comment by rtivital** [comment]
We will consider adding this feature

**Comment by Jochem-Groeneweg** [comment]
Any updates?

**Reply by rtivital** [reply]
Will be available in 7.0 for Badge and all other components.
